Libraries/Inspector/NetworkOverlay.js

```diff
@@ -10,7 +10,7 @@
 
 'use strict';
 
-import type {RenderItemProps} from '../Lists/VirtualizedList';
+import type {RenderItemProps} from '@react-native/virtualized-lists/Lists/VirtualizedList';
```

Comment:
For this and other examples, we should be using the public exports instead of reaching inside.

Reply:
Sure, updated